src/db/queryExecuter.ts

```diff
@@ -0,0 +1,9 @@
+export const executeQuery = async (client: any, query: string) => {
```

Comment:
it's not query in our system though, it should be named accordingly

Reply:
we dont have query anyway, we call it dataSourceId, but I will change